app/models/analytics/AnalyticsService.scala

```diff
@@ -1,49 +1,66 @@
 package models.analytics
 
 import com.scalableminds.util.accesscontext.{DBAccessContext, GlobalAccessContext}
-import com.scalableminds.util.time.Instant
 import com.scalableminds.util.tools.Fox
+import com.scalableminds.util.tools.Fox.bool2Fox
 import com.scalableminds.webknossos.datastore.rpc.RPC
 import com.typesafe.scalalogging.LazyLogging
-
-import javax.inject.Inject
-import models.annotation.Annotation
-import models.dataset.{Dataset, DataStore}
-import models.organization.Organization
-import models.user.{MultiUserDAO, User, UserDAO}
-import models.job.JobCommand.JobCommand
+import models.user.{MultiUserDAO, UserDAO}
 import play.api.libs.json._
 import utils.{ObjectId, WkConf}
 
+import javax.inject.Inject
 import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 
 class AnalyticsService @Inject()(rpc: RPC,
                                  wkConf: WkConf,
                                  analyticsLookUpService: AnalyticsLookUpService,
-                                 analyticsSessionService: AnalyticsSessionService)(implicit ec: ExecutionContext)
+                                 analyticsSessionService: AnalyticsSessionService,
+                                 analyticsDAO: AnalyticsDAO)(implicit ec: ExecutionContext)
     extends LazyLogging {
 
+  private lazy val conf = wkConf.BackendAnalytics
+  private lazy val wellKnownUris = conf.wellKnownUris.map(_.split("\\|")).map(parts => (parts(0), parts(1))).toMap
```

Comment:
should probably be wrapped in a tryo to prevent exceptions during initialization of this class. When using the result, match on the box state